biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -491,7 +492,29 @@ public interface Constants {
 	 */
 	String		DEFAULT_PREPROCESSS_MATCHERS				= "!*.(ico|jpg|jpeg|jif|jfif|jp2|jpx|j2k|j2c|fpx|png|gif|swf|doc|pdf|tiff|tif|raw|bmp|ppm|pgm|pbm|pnm|pfm|webp|zip|jar|gz|tar|tgz|exe|com|bin|mp[0-9]|mpeg|mov|):i, *";
 
-	/*
+	/**
+	 * Headers that if **all** are absent will trigger the -includepackage
+	 * `*;from:=project` (include all packages from the project's output.
+	 */
+	String[]	EXPAND_HEADERS							= {
+		Constants.RESOURCEONLY, Constants.INCLUDEPACKAGE, Constants.PRIVATE_PACKAGE, Constants.PRIVATEPACKAGE,
+		Constants.EXPORT_PACKAGE,
+	};
```

Comment:
```suggestion
	Set<String>	EXPAND_HEADERS							= Sets.of(
		Constants.RESOURCEONLY, Constants.INCLUDEPACKAGE, Constants.PRIVATE_PACKAGE, Constants.PRIVATEPACKAGE,
		Constants.EXPORT_PACKAGE,
	);
```

We have moved to use unmodifiable sets for such groups in Constants. This prevents mistakes like duplicates and mutation.